src/librustc_errors/emitter.rs

```diff
@@ -856,18 +858,28 @@ impl EmitterWriter {
     fn fix_multispans_in_std_macros(&mut self,
                                     span: &mut MultiSpan,
                                     children: &mut Vec<SubDiagnostic>,
+                                    level: &Level,
                                     backtrace: bool) {
         let mut spans_updated = self.fix_multispan_in_std_macros(span, backtrace);
         for child in children.iter_mut() {
             spans_updated |= self.fix_multispan_in_std_macros(&mut child.span, backtrace);
         }
+        // here
```

Comment:
What is the purpose of this comment?

Reply:
Oops, I forgot removing the mark for myself, I'll remove :)